cdap-master-spi/src/main/java/io/cdap/cdap/master/spi/environment/MasterEnvironmentContext.java

```diff
@@ -24,6 +24,7 @@
  * Context object available to {@link MasterEnvironment} for access to CDAP resources.
  */
 public interface MasterEnvironmentContext {
+  String ENVIRONMENT_PROPERTY_PREFIX = "environment.property.";
```

Comment:
nit: add comments explaining what this prefix is and how it will be used

Reply:
Added.